SU-SWS/Stanford-Drupal-Profile
# Add node_revision_delete

# READY FOR REVIEW


# Summary
- Adds [Node Revision Delete](https://drupal.org/project/node_revision_delete)

# Needed By (Date)
- The horse has already left the barn (it's already been added to gitolite)

# Criticality
- How critical is this PR on a 1-10 scale? 1

# Steps to Test

1. Run `drush make make/group.make /path/to/group-sitedir`
2. Run `drush make make/dept.make /path/to/dept-sitedir`
3. Ensure that the `node_revision_delete` module exists in `sites/all/modules/contrib` in both Drupal docroots

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? It helps with trimming down database size due to runaway Course revisions.

# Associated Issues and/or People
- Other PRs: https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/168
- Anyone who should be notified? ( @pookmish )

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
